### PR TITLE
Documentation refinement

### PR DESCRIPTION
**Pull request by cooperlab**
Pull request done for line comments

### CODE REVIEW

File: docs/example.rst [hunk 1]
```diff
@@ -1,97 +1,58 @@
...
-Go to http://localhost/HistomicsML/reports.html?application=nuclei
-
-.. image:: images/report-1.png
+At the home page under *Continue a session*, select the dataset and training set name and click ``Continue``. Navigate to the ``Review`` interface by clicking the tab at the top menu.
 
-* Slide class counts: counts (numbers) of samples that fall into different categories. For example, the numbers of positive and negative samples in a slide. (.csv)
-* Map scores: score, centroid X, and centroid Y for each sample on a slide. (.csv)
-* Download a training set: a training set. (.h5)
-* Apply classifier: scores for each prediction. Predicted scores will be added to the current dataset. (.h5)
+The review interface displays the annotated objects organized by class and slide. Thumbnail images of the objects are organized into columns by class. Clicking a thumbnail will bring that object into the field of view in the slide view. The thumbnails can be dragged/dropped to a different column to change the class label, or placed in the ``Ignore`` column to discard them from the set. Changes are instantly commited to the database (no additional button clicks are needed).
```
Comment (cooperlab) [inline docs/example.rst]: @slee172 Are changes made in Review by dragging thumbnails instantly committed to the database? Do we need to use retrain (for training) or save (for validation) to commit these changes?

Comment (slee172) [reply docs/example.rst]: @cooperlab Changes are not going to the database directly. Instead, they stay on the memory of the learner server. Once the users finalize their training on the instance tab, all the training samples are stored in the database.

File: docs/data-format.rst [hunk 2]
```diff
@@ -1,88 +1,75 @@
...
-  /mean -  Mean value of each feature. (Used for z-score normalization)
-  /std_dev - Standard deviation of each feature. (Used for z-score normalization)
-  /slides	-	Names of the slides/images in the dataset
-  /slideIdx - Index into the slides data for each sample, 0 is the first slide, 1 the sceond...
-  /x_centroid - Float, x location of the sample in the image.
-  /y_centroid - Float, y location of the sample in the image.
-
-You can see our sample file (GBM-features.h5) provided in the docker container,
-following the command.
+  /features - A D x N array of floats containing the feature values for each object in the dataset (N objects, each with D features). Each feature/row should be normalized by z-score.
```
Comment (slee172) [inline docs/data-format.rst:63]: @cooperlab This should be N (row) x D (column) array (N objects, D features)

File: docs/installation.rst [hunk 3]
```diff
@@ -90,23 +81,27 @@ Now, we describe how to install HistomicsML using docker container.
...
-8. Start learning server:
+7. Start the server
 
 .. code-block:: bash
 
  root@97d439b58033:/var/www/html/HistomicsML/db# service al_server start
  Starting active learning server daemon al_server [ OK ]
  root@97d439b58033:/var/www/html/HistomicsML/db# exit
+ 
+.. note:: If the server becomes unresponsive or generates a connection error during use, the al_server will need to be restarted.
```
Comment (slee172) [inline docs/installation.rst:105]: @cooperlab In order to start the al_server again, 1. stop the al_server using "service al_server stop" 2. wait for a few seconds for the system ready 3. start the al_server using "service al_server start".

**Pull request merged**